build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,43 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
+MAINTAINER Simon Bennetts "psiinon@gmail.com"
+
+WORKDIR /zap
+
+RUN apt-get update && apt-get install -q -y --fix-missing \
+	wget \
+	curl \
+	xmlstarlet \
+	unzip && \
+	apt-get clean && \
+	rm -rf /var/lib/apt/lists/*
+
+# Download and expand the latest stable release 
+RUN curl -s https://raw.githubusercontent.com/zaproxy/zap-admin/master/ZapVersions-dev.xml | xmlstarlet sel -t -v //url |grep -i Linux | wget --content-disposition -i - -O - | tar zxv && \
```

Comment:
It was also weird to me, but I noticed that the stable image is using the dev from some reason - so I assumed this is something I am missing.